src/main/java/org/opentripplanner/datastore/file/ZipFileDataSource.java

```diff
@@ -93,6 +96,30 @@ private void loadContent() {
         ZipEntry entry = entries.nextElement();
         content.add(new ZipFileEntryDataSource(this, entry));
       }
+    } catch (ZipException ze) {
```

Comment:
Maybe useful to add a log message at INFO level, that might help in production environments.